pyipn/geometry.py

```diff
@@ -47,6 +47,9 @@ def get_light_travel_time(self, other_location):
     def coord(self):
 
         return self._skycoord
+    
+    def get_cartesian_coord(self):
+        return self._skycoord.represent_as(CartesianRepresentation)
```

Comment:
Does this return a copy or the coord itself?

Reply:
Good question. I believe represent_as() returns a copy but I'm not sure